crates/figma_import/src/fetch.rs

```diff
@@ -118,19 +118,22 @@ pub fn fetch_doc(
             doc.component_sets().clone(),
             variable_map,
         );
-        let mut response = bincode::serialize(&DesignComposeDefinitionHeader::current(
+
+        let header = DesignComposeDefinitionHeader::current(
             doc.last_modified().clone(),
             doc.get_name(),
             doc.get_version(),
             doc.get_document_id(),
-        ))?;
-        response.append(&mut bincode::serialize(&ServerFigmaDoc {
+        );
+        let server_doc = ServerFigmaDoc {
             figma_doc: Some(figma_doc),
             errors: error_list,
             branches: doc.branches.clone(),
             project_files: vec![],
-        })?);
+        };
 
+        let mut response = header.encode_length_delimited_to_vec();
+        response.append(&mut server_doc.encode_length_delimited_to_vec());
```

Comment:
Can you just call `encode_dcd_with_header()` instead

Reply:
Not in this case because it's the ServerFigmaDoc and I didn't make that function for it. This is the only place we do it so I don't think it's worth it here.